cot/src/json.rs

```diff
@@ -0,0 +1,21 @@
+//! JSON rendering utilities.
+//!
+//! This module provides structures and methods for creating and rendering JSON
+//! content.
+
+/// A type that represents JSON content.
+///
+/// Note that this is just a newtype wrapper around data and does not
+/// provide any content validation.
```

Comment:
I would mention that this is primarily useful as a request extractor and response type for RESTful endpoints.